index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css">
+    <title>Freddy KABERUKA</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="top-nav">
+            <div class="logo">
+                <h1>FREDDY K</h1>
+            </div>
+            <div>
+                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" class="logo-info" viewBox="0 0 24 24">
+                    <path d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z" />
+                </svg>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="personal-info">
+            <div class="personal-info-wrap">
+                <div class="personal-info-title">
+              <h2><a href="#" class="header-info">I'm Freddy Glad to see you!</a></h2>
+            </div>
+            <div class="personal-info-detail">
+                <p>I'm a software developer i can help you build a product a product. feature
+                or website Look through some of my work and experience! if ypu like what you see and have
+                a project you need coded don't hestate to contact me
+               </p>
+            </div>
+            </div>
+            
+            <div class="personal-social-info">
+                <h3>Let's CONNECT</h3>
+                <div class="social-media">
+                    <ul>
+                    <li><a href="#"><img src="/assets/twitter.svg" alt="twitter"/></a></li>
+                    <li><a href="#"><img src="/assets/in.png" alt="linkdin"></a></li>
+                    <li><a href="#"><img src="/assets/m.svg" alt=""></a></li>
+                    <li><a href="#"><img src="/assets/git.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="/assets/peace.svg" alt="peace"></a></li>
```

Comment:
- [ ] Please fix your image path `src="/assets/twitter.svg" ` it doesn't display

### **Correct Path**
`src="assets/twitter.svg" `